src/harness/util.py

```diff
@@ -125,3 +135,14 @@ def makePpaAndPalRecordsConsistent(ppa_record, pal_records, low_frequency,
   # Converting from defaultdict to dict
   ppa_record = json.loads(json.dumps(ppa_record))
   return ppa_record, pal_records
+
+
+def TriggerDailyActivitiesImmediatelyAndWaitUntilComplete(_sas_admin):
```

Comment:
I think I agree with you in a way that if the functions need an access to member variables we can have that function whether it's utility or assertion in the sas_testcase.py. But if it doesn't require any access to member variables then we can keep that in the util.py?